documentation/contributing/index.markdown

```diff
@@ -0,0 +1,108 @@
+---
+author: davetcoleman
+comments: true
+date: 2016-7-21 02:13:26+00:00
+layout: page
+slug: tutorials
+title: Contributing
+---
+
+# Contributing to MoveIt!
+
+We want to encourage all MoveIt! users to contribute back to the open source project, whether enhancing documentation, answer user questions, reporting & fixing bugs, reviewing pull requests, or adding new features.
+
+## Enhancing Documentation
+
+Documentation for the MoveIt! project can be found in two places: within the source repositories and on the main website:
+
+### Within the source repositories as Sphinx files
+
+Tutorials and other concepts are documented as [reStructredText](http://docutils.sourceforge.net/rst.html) files (similar to Markdown) in ``doc`` folders in various MoveIt! packages. For example, many of the beginner tutorials use the PR2 robot as an example, and their documentation pages can be found [here](https://github.com/ros-planning/moveit_pr2/tree/indigo-devel/pr2_moveit_tutorials/planning/src/doc)
+
+To edit or add more tutorials, create Github pull requests to the appropriate location and the maintainers will graciously approve your help after review.
+
+If you are interested in generating the html pages locally on your machine, install [rosdoc_lite](http://wiki.ros.org/rosdoc_lite) (``apt-get install ros-kinetic-rosdoc-lite``) and run in the root of the package:
+
+    rosdoc_lite .
+
+i.e. [pr2_moveit_tutorials](https://github.com/ros-planning/moveit_pr2/tree/indigo-devel/pr2_moveit_tutorials). This will automatically pick up the rosdoc.yaml and write the documentation to ``doc/html``
+
+For deploying documentation changes to the web, note [Section 3 of rosdoc_lite wiki](http://wiki.ros.org/rosdoc_lite):
+
+>  rosdoc_lite is automatically run for packages in repositories that have rosinstall files listed in the rosdistro repository. The resulting documentation is uploaded to ros.org and is linked in the "Code API" links that you see on various package pages, like rospy.
+
+This is done about once every 24 hours, [overnight](http://wiki.ros.org/rosdistro/Tutorials/Indexing%20Your%20ROS%20Repository%20for%20Documentation%20Generation).
+
+### moveit.ros.org
+
+Once you have created new Sphinx documentation, you'll need to create links to it from the main moveit.ros.org website so that other users can find it. Additionally, some higher level documentation can be found directly on this website, so you'll need to edit it there as well.
+
+To edit or add pages on the MoveIt! website, go to [https://github.com/ros-planning/moveit.ros.org](https://github.com/ros-planning/moveit.ros.org) and either edit inline using Github's web-based editing capability, or ``git clone`` the repo and push pull requests back to the repo.
```

Comment:
On Wed, Jul 27, 2016 at 09:33:50AM -0700, Dave Coleman wrote:
> I did but after long discussions with OSRF (@tfoote) I was blocked by @sachinchitta 

For what reason? In my opinion it doesn't make much sense to have the website documentation at one
individual's account. This confuses even devs (it confused me), users (potential documentation writers) much more so, I'm afraid.
Especially, because very few people know the nick "julienking"...